### PR TITLE
Frontend/dashboard

### PR DESCRIPTION
**Pull request by AndrewDucDanhDo**
Fix bugs with create task

### CODE REVIEW

**Review by CalvinDong — CHANGES_REQUESTED** [review]
- unable to verify if the view task page works with the backend.
- receiving 404 error when using master database, 
- tested on user: test123, task: 608d0ed09d4ba62d14a5e9dc
- mongo error at task controller: CastError: Cast to ObjectId failed for value "recipe01" at path "_id" for model "Recipe", Argument passed in must be a single String of 12 bytes or a string of 24 hex characters

**Comment by CalvinDong** [comment]
what database was this working on?

**Comment by AndrewDucDanhDo** [comment]
been using the beansprouts database - will try with master database

File: backend/src/model/task.ts [hunk 1]
```diff
@@ -20,10 +20,6 @@ const taskSchema = new mongoose.Schema({
     archive: {
         type: Boolean,
         required: false
-    },
-    type: {
-        type: String,
-        required: false
```
Comment (mitcmurph) [inline backend/src/model/task.ts:26]: Is this suppose to be removed

**Comment by mitcmurph** [comment]
> * unable to verify if the view task page works with the backend.
> * receiving 404 error when using master database,
> * tested on user: test123, task: 608d0ed09d4ba62d14a5e9dc
> * mongo error at task controller: CastError: Cast to ObjectId failed for value "recipe01" at path "_id" for model "Recipe",
> * Argument passed in must be a single String of 12 bytes or a string of 24 hex characters

Looks like bad data.

**Comment by CalvinDong** [comment]
> > * unable to verify if the view task page works with the backend.
> > * receiving 404 error when using master database,
> > * tested on user: test123, task: 608d0ed09d4ba62d14a5e9dc
> > * mongo error at task controller: CastError: Cast to ObjectId failed for value "recipe01" at path "_id" for model "Recipe",
> > * Argument passed in must be a single String of 12 bytes or a string of 24 hex characters
> 
> Looks like bad data.

That's probably it. Works on beansprouts

File: backend/src/model/task.ts (hunk 1 above)
Comment (mitcmurph) [reply backend/src/model/task.ts:26]: Unrelated to changes - caused by merge

**Review by mitcmurph — APPROVED** [review]

**Review by mitcmurph — APPROVED** [review]

**Review by CalvinDong — APPROVED** [review]
very nice, very cool dashboard

**Pull request merged**